synapse/config/server.py

```diff
@@ -486,6 +486,15 @@ def read_config(self, config: JsonDict, **kwargs: Any) -> None:
         else:
             self.redaction_retention_period = None
 
+        # How long to keep locally forgotten rooms before purging them.
+        purge_retention_period = config.get("purge_retention_period", "7d")
```

Comment:
Can we call this something like `forgotten_room_retention_period`? I think it's important to include that in the name.

Can you also add the new option to the docs please?